src/sagemaker_inference/model_server.py

```diff
@@ -159,7 +159,7 @@ def _generate_mms_config_properties(env, handler_service=None):
         "default_workers_per_model": env.model_server_workers,
         "inference_address": "http://0.0.0.0:{}".format(env.inference_http_port),
         "management_address": "http://0.0.0.0:{}".format(env.management_http_port),
-        "vmargs": "-XX:-UseContainerSupport",
+        "vmargs": env.vmargs,
```

Comment:
@davidthomas426 That's correct, vmargs specified in config file was ignored in old code, and will be ignored in the new code as well. Since `env.vmargs` is set after reading config file, it should override any previous value, if specified at all, else will use default value.